inputgraph/loader.go

```diff
@@ -22,7 +23,10 @@ import (
 	"github.com/pkg/errors"
 )
 
-var ErrRemoteNotSupported = fmt.Errorf("remote targets not supported")
+var (
+	errRemoteNotSupported  = errors.New("remote targets not supported")
```

Comment:
Is this error still needed?

Reply:
Not necessarily, but it is used below to guard the non-remote code path against passed-in remote targets.

Reply:
I clarified the name a bit.